### PR TITLE
EWL-549:add story card component

### PR DESCRIPTION
**Pull request by val3ntin**
* no mobile/ tablet design here.
the card has 100% width so it's responsive by default.
thus, once you use it inside the Grid component it obviously take the place needed....

design match on 343px width:
Screen Shot 2019-04-24 at 10 38 52 AM

### CODE REVIEW

File: src/components/StoryCard/index.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    position: absolute;
+    top: 0;
+    width: 100%;
+    height: auto;
+  }
+`;
+
+const StyledContent = styled.div`
+  position: relative;
+  padding: 39px 30.17px 0 26.81px;
```
[the remark this answers is not in the document]
Comment (val3ntin) [reply src/components/StoryCard/index.js]: yes, you're right

[the remark this answers is not in the document]
Comment (Xb0X) [reply src/components/StoryCard/index.js]: 👍

[the remark this answers is not in the document]
Comment (val3ntin) [reply src/components/StoryCard/index.js]: will ceil to the closest int

[the remark this answers is not in the document]
Comment (val3ntin) [reply src/components/StoryCard/index.js]: done

**Pull request merged**